subprojects/docs/src/main/resources/header.html

```diff
@@ -197,11 +197,6 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
             <li><a class="nav-dropdown" data-toggle="collapse" href="#cpp-projects" aria-expanded="false" aria-controls="cpp-projects">C++ Projects</a>
                 <ul id="cpp-projects">
                     <li><a href="cpp_plugins.html">Building C++ projects</a></li>
-                    <li><a href="native_software.html">Building other native language projects</a></li>
-                    <li><a href="software_model_concepts.html">Software Model Concepts</a></li>
-                    <li><a href="software_model.html">Rule-based Model Configuration</a></li>
-                    <li><a href="rule_source.html">Implementing Model Rules in a Plugin</a></li>
-                    <li><a href="software_model_extend.html">Extending the Software Model</a></li>
```

Comment:
Since only the first of these chapters — native_software.html — is linked from the new C++ chapter, would you mind making sure native_software.adoc links to the other chapters somewhere? I saw that it already links to Software Model Concepts, but not sure about the others.

Reply:
You can discover all the other chapter from the native_software.html.